Client/Frontend/Browser/PasswordHelper.swift

```diff
@@ -82,6 +82,11 @@ class PasswordHelper: BrowserHelper {
     }
 
     private func setPassword(password: Password) {
```

Comment:
This is not really a comment on this change, but I find the name `Password` a bit odd here. Specially with `Password.password`. Maybe at one point we can rename it to `Credentials` or `Login`. maybe?